pkgs/servers/code-server/default.nix

```diff
@@ -43,12 +43,43 @@ in stdenv.mkDerivation rec {
     '';
   };
 
+  esbuild-code-server-build = buildGoModule rec {
+    pname = "esbuild";
+    version = "0.12.6";
+
+    src = fetchFromGitHub {
+      owner = "evanw";
+      repo = "esbuild";
+      rev = "v${version}";
+      sha256 = "1myc4z2cn32qa9bzgyvn1schpjswc3jvy29mxm9pbqa6ijqlgi4x";
+    };
+
+    vendorSha256 = "1n5538yik72x94vzfq31qaqrkpxds5xys1wlibw2gn2am0z5c06q";
+  };
+
+  esbuild-code-server-extensions = buildGoModule rec {
+    pname = "esbuild";
+    version = "0.11.12";
+
+    src = fetchFromGitHub {
+      owner = "evanw";
+      repo = "esbuild";
+      rev = "v${version}";
+      sha256 = "1mxj4mrq1zbvv25alnc3s36bhnnhghivgwp45a7m3cp1389ffcd1";
+    };
+
+    vendorSha256 = "1n5538yik72x94vzfq31qaqrkpxds5xys1wlibw2gn2am0z5c06q";
+  };
+
   yarnCache = stdenv.mkDerivation {
     name = "${pname}-${version}-${system}-yarn-cache";
     inherit src;
-    nativeBuildInputs = [ yarn' git ];
+    nativeBuildInputs = [ yarn' git cacert ];
     buildPhase = ''
       export HOME=$PWD
+      export GIT_SSL_CAINFO="${cacert}/etc/ssl/certs/ca-bundle.crt"
```

Comment:
That shouldn't be required.

Reply:
Without this, the following error presents when running `nix-build -A code-server --pure`:

```
error Command failed.
Exit code: 128
Command: git
Arguments: ls-remote --tags --heads https://github.com/mjbvz/markdown-it-katex.git
Directory: /build/source
Output:
fatal: unable to access 'https://github.com/mjbvz/markdown-it-katex.git/': SSL certificate problem: unable to get local issuer certificate
```

Is there a better solution for this error?